docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.7'
+
+services:
+  app:
+    image: ruby:2.6
+    volumes:
+      - .:/app
+    working_dir: /app
+    command: bash
+    depends_on:
+      - redis
+    environment:
+      REDIS_HOST: redis
+      BUNDLE_PATH: /app/.gems
+
+  redis:
+    image: redis:latest
```

Comment:
What about setting an specific Redis version and not use `latest` ?

This way, helps to identify what is the supported version of Redis and it will not break when there is a major upgrade in Redis.

Reply:
Sure! I'm going to lock it in 5.0.6 - which seems to be the latest one on dockerhub, okay?

Reply:
good!